### PR TITLE
[WIP] MDCT-478: Dashboard Implementation

### PR DESCRIPTION
**Pull request by ntsummers1**
## Description
💨 Draft

### How to test
./dev local
Go to mcpar/dashboard

## Code author checklist
- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/components/cards/TemplateCard.tsx [hunk 1]
```diff
@@ -115,6 +115,7 @@ const sx = {
   },
   actionsFlex: {
     flexFlow: "wrap",
+    gridGap: "1rem",
```
[the remark this answers is not in the document]
Comment (gmrabian) [reply services/ui-src/src/components/cards/TemplateCard.tsx:118]: Good spacing fix!

File: services/ui-src/src/forms/mcpar/dash/dashForm.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      "id": "dash-title",
+      "type": "text",
+      "props": {
+        "label": "Program name"
+      }
+    },
+    {
+      "id": "dash-contractPeriod",
+      "type": "dropdown",
+      "props": {
```
[the remark this answers is not in the document]
Comment (gmrabian) [reply services/ui-src/src/forms/mcpar/dash/dashForm.json:17]: Might kick this to later

File: services/ui-src/src/components/reports/ReportProvider.tsx [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
   writeReport,
   deleteReport,
 } from "utils";
 // verbiage
 import { reportErrors } from "verbiage/errors";
 
 export const ReportContext = createContext<ReportContextShape>({
-  reportStatus: undefined as AnyObject | undefined,
+  report: undefined as AnyObject | undefined,
   reportData: undefined as AnyObject | undefined,
```
Comment (braxex) [inline services/ui-src/src/components/reports/ReportProvider.tsx:22]: Eventually we should firm these types up but I'm not concerned about it for right now.

Comment (gmrabian) [reply services/ui-src/src/components/reports/ReportProvider.tsx:22]: agreed

**Review by braxex — APPROVED** [review]

**Comment by braxex** [comment]
Note: The very slight drop in code coverage is ok with me.

**Pull request merged**